### DISCUSSION TITLE
Install the Company Portal app for MacOS as a MacOS LOB app

### DISCUSSION THREAD

**Discussion by giscus** [ugurkocde/intunemacadmins#49] [General]
# Install the Company Portal app for MacOS as a MacOS LOB app

Step-by-step instructions on installing the Company Portal app for MacOS as a Line-of-Business (LOB) app. Learn how to download, add, and assign the app in Intune for seamless device management.

https://www.intunemacadmins.com/complete-guide-macos-deployment/install_the_company_portal_app_for_macos_as_a_macos_lob_app/

**Comment by theITcloudAdmn** [comment]
Maybe it's a good idea to note, that the list of included apps must be drastically shortened to not get an installation error. I only include this two apps for a successful install:
- com.microsoft.CompanyPortalMac
- com.microsoft.autoupdate

**Comment by marcoheijkoop** [comment]
Company Portal for macOS location is changed. https://go.microsoft.com/fwlink/?linkid=853070
